runtime/parachains/src/runtime_api_impl/v1.rs

```diff
@@ -231,6 +232,21 @@ pub fn session_index_for_child<T: initializer::Config>() -> SessionIndex {
 	<shared::Module<T>>::session_index()
 }
 
+pub fn get_historical_authority_ids<T: initializer::Config>() -> Vec<AuthorityDiscoveryId> {
+	let current_session_index = session_index_for_child::<T>();
+	let earliest_stored_session = <session_info::Module<T>>::earliest_stored_session();
+	let mut authority_ids = vec![];
+
+	for session_index in earliest_stored_session..current_session_index {
+		let info = <session_info::Module<T>>::session_info(session_index);
+		if info.is_some() {
+			authority_ids.append(&mut info.unwrap().discovery_keys);
+		}
```

Comment:
we have a policy of no `unwrap`s, esp. in the runtime code
https://github.com/paritytech/substrate/blob/master/docs/STYLE_GUIDE.md#style
```suggestion
		if let Some(info) = info {
			authority_ids.append(&mut info.discovery_keys);
		}
```